<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Snap ITC</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>배추PC방</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="NUM">
       <property name="font">
        <font>
         <family>Snap ITC</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>NO.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="SeatLabel">
       <property name="font">
        <font>
         <family>휴먼모음T</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="BEGIN">
       <property name="font">
        <font>
         <family>휴먼모음T</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>*시작시간</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="UsageLabel">
       <property name="font">
        <font>
         <family>휴먼모음T</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="REMAIN">
       <property name="font">
        <font>
         <family>휴먼모음T</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>*남은시간</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="BeginLabel">
       <property name="font">
        <font>
         <family>휴먼모음T</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="USAGE">
       <property name="font">
        <font>
         <family>휴먼모음T</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>*사용시간</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="RemainLabel">
       <property name="font">
        <font>
         <family>휴먼모음T</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ID">
       <property name="font">
        <font>
         <family>휴먼모음T</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>*아이디</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLabel" name="IdLabel">
       <property name="font">
        <font>
         <family>휴먼모음T</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>431</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLineEdit" name="MsgLineEdit">
       <property name="font">
        <font>
         <family>한컴 백제 B</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="MsgBtn">
       <property name="font">
        <font>
         <family>한컴 백제 B</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>전송</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="MsgListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한컴 백제 B</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="craft">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="3rd.qrc">:/example/craft.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pubg">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>240</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="3rd.qrc">:/example/pubg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="maple">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="3rd.qrc">:/example/maple.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lol">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="3rd.qrc">:/example/lol.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>양재매화체S</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>121</width>
     <height>106</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="3" column="0">
     <widget class="QPushButton" name="CloseBtn">
      <property name="font">
       <font>
        <family>MD개성체</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>사용종료</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="ShiftBtn">
      <property name="font">
       <font>
        <family>MD개성체</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string> 자리이동</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="FoodBtn">
      <property name="font">
       <font>
        <family>MD개성체</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>먹거리 상품주문</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="3rd.qrc"/>
 </resources>
 <connections/>
</ui>
